content/guides/metadata/1-templates/0-index.md

```diff
@@ -0,0 +1,34 @@
+---
+alias_paths: []
+---
+
+# Metadata templates
+
+A [metadata template][template] describes a set of key/value
+pairs that can be assigned to a file or folder.
+
+For example, an `invoiceData` template might hold data about an invoice, having
+a field for the invoice ID as well as the customer ID.
+
+A file or folder can have multiple distinct template [instances][instance]
+associated with it, such as a `marketingCollateral` instance and a
+`retentionPolicy` instance.
+
+## Metadata scopes
+
+Metadata templates are grouped into two distinct groups, or **scopes**.
+
+<CTA to='g://metadata/templates/scopes'>
+  Learn more about Metadata Scopes
+</CTA>
+
+## Restrictions
+
+There is a limit of 500 templates per enterprise.
```

Comment:
Do we need an extra callout here for who/the required permissions to make templates?

Reply:
Good point, I'll do